model/infinispan/src/main/java/org/keycloak/connections/infinispan/InfinispanConnectionProvider.java

```diff
@@ -43,7 +43,7 @@
     String WORK_CACHE_NAME = "work";
     String AUTHORIZATION_CACHE_NAME = "authorization";
     String AUTHORIZATION_REVISIONS_CACHE_NAME = "authorizationRevisions";
-    int AUTHORIZATION_REVISIONS_CACHE_DEFAULT_MAX = 20000;
+    int AUTHORIZATION_REVISIONS_CACHE_DEFAULT_MAX = 1000000;
```

Comment:
I am not sure I have enough knowledge of the Infinispan layer to review this. @hmlnarik @slaskawi Could you please help me on this line?

Reply:
This is only changing the defaults using a higher value for the number of entries in cache. Useful when you have lot of authorization related entries.

But I'm going to revert this change though, people should be able to change that value. I used that during tests ...